### DISCUSSION TITLE
Master's project

### DISCUSSION THREAD

**Discussion by petravanhoudt** [OSIPI/DCE-DSC-MRI_CodeCollection#120] [Ideas]
At the meeting last week, the idea came up to create a master project for some of the tasks in the new roadmap. 
I was discussing this with my husband who studied software engineering himself. He suggested that this might be better suited for a bachelor student. For a master student it requires a research component, which this project might not have from a software point of view. Just wanted to share this insight, at least it wasn't something I had thought about.

**Comment by stadmill** [comment]
Thanks for this Petra. You are right, it is well suited for an undergraduate project. @LucyKershaw and @mjt320 mentioned in the meeting that they might be able to find out more about such cross-faculty projects. The only thing is we will need to have really clear design specs before we can offer it as a student project. Bringing a potential supervisor from a Computer Science (or similar) department into the TF will help us scope the project properly too.

**Reply by LucyKershaw** [reply]
Michael has been in touch with someone who is in charge of projects like that, so we'll see what happens. We also asked that the project be supervised jointly by someone with more relevant background knowledge(!)
